doofinder/includes/api/class-store-api.php

```diff
@@ -161,13 +159,87 @@ private function sendRequest($endpoint, $body)
         }
     }
 
+    /**
+     * Generates the create-store payload
+     *
+     * @param array $api_keys The list of search engine ids
+     * @return void
+     */
+    public function build_store_payload($api_keys)
+    {
+        $primary_language = get_locale();
+        if ($this->language->get_languages() != null) {
+            $primary_language = $this->language->get_base_language();
+        }
+
+        $primary_language = Helpers::format_locale_to_hyphen($primary_language);
+        $domain = str_ireplace('www.', '', parse_url(get_bloginfo('url'), PHP_URL_HOST));
 
-    public function format_language_code($code)
+        $store_options =  self::get_store_options();
+        if (is_null($store_options)) {
+            throw new Exception("Invalid store options");
+        }
+
+        $callback_urls = self::get_callback_urls($api_keys, $primary_language);
+        $currency = is_plugin_active('woocommerce/woocommerce.php') ? get_woocommerce_currency() : "EUR";
+
+        $store_payload = [
+            "name" =>  get_bloginfo('name'),
+            "platform" => "wordpress",
```

Comment:
I seem to recall that there was talk because the Store platform had to be dynamic. If you have woocommerce installed you put that one. And if not, wordpress.

Has that changed?

Reply:
You're right, thanks, I'll fix it now